redash/handlers/data_sources.py

```diff
@@ -141,3 +141,15 @@ def delete(self, data_source_id):
         })
 
         return data_source.to_dict()
+
+
+class DataSourceTestResource(BaseResource):
+    @require_admin
+    def post(self, data_source_id):
+        data_source = get_object_or_404(models.DataSource.get_by_id_and_org, data_source_id, self.current_org)
+        try:
+            data_source.query_runner.test_connection()
+        except Exception, e:
+            return {"msg": "failure: {}".format(e), "ok": 0}
```

Comment:
Why not use `True`/`False` instead of 0/1?

And let's use `message` instead of `msg` to be consistent with other responses.